packages/generator/src/edmx-to-vdm/edmx-to-vdm-util.ts

```diff
@@ -111,7 +116,14 @@ export function typesForCollection(
       fieldType: 'CollectionField'
     };
   }
-  if (isComplexType(typeInsideCollection)) {
+  if (isComplexTypeOrEnumType(typeInsideCollection)) {
```

Comment:
Also here I would make it symmetric between enum and complex type:
```
if(isEnumType(string,enums)){
return enumStuff
}
if(isComplex(string,complexTypes)){
return ComplexStuff
}

thorw execption cause nothing is matched.
```

Reply:
The caller (and the caller of the caller) of this function is creating the `complexTypes`, that's why you have to determine `isComplex` by using `isComplexTypeOrEnumType` and `isEnum`